src/dra/5298C.c

```diff
@@ -1411,7 +1411,35 @@ void func_800F9DD0(u8* arg0, u8* arg1) {
 
 INCLUDE_ASM("dra/nonmatchings/5298C", func_800F9E18);
 
+#if defined(VERSION_HD)
 INCLUDE_ASM("dra/nonmatchings/5298C", func_800F9F40);
+#else
+void func_800F9F40(void) {
+    Unkstruct_800F9F40* var_a2;
+    Unkstruct_800F9F40* var_a3;
+    Unkstruct_800F9F40* ptr;
+    char buffer[38];
+    s32 idx;
+    u8 spellId;
+    s32 i;
+
+    for (i = 0, ptr = D_800DC70C, idx = 0x80; i < 8; i++) {
+        spellId = D_80097902[idx];
+        if (spellId & 0x80) {
+            spellId ^= 0x80;
+            var_a3 = (Unkstruct_800F9F40*)buffer;
+            var_a2 = (Unkstruct_800F9F40*)&D_800DC6EC;
+            do {
+                __builtin_memcpy(var_a3++, var_a2++, 0x10);
```

Comment:
Could be                 
```
__builtin_memcpy(var_a3++, var_a2++, sizeof(Unkstruct_800F9F40));
```
Not sure what's going on with the 0xB

Reply:
Probably the struct needs to be the right size, but again, it's weird because they are using the same struct and the number is different in both instances, i don't understand the function enough to tell if they hardcoded those values or we have two different structs there, which sounds to me is very unlikely.